src/server/Worker.ts

```diff
@@ -350,6 +356,31 @@ export interface TextLanguageDetectionResponseMessage extends WorkerMessageBase,
 	messageType: 'TextLanguageDetectionResponse'
 }
 
+///////////////////////////////////////////////////////////////////////////////////////////////
+// Voice activity detection operations
+///////////////////////////////////////////////////////////////////////////////////////////////
+async function processVoiceActivityDetectionRequest(message: VoiceActivityDetectionRequestMessage, sendMessage: MessageFunc) {
+	const result = await detectVoiceActivity(message.input, message.options)
+
+	const responseMessage: VoiceActivityDetectionResponseMessage = {
+		messageType: 'VoiceActivityDetectionResponse',
+		...result
+	}
+
+	sendMessage(responseMessage)
+}
+
+// Voice activity detection message types
+export interface VoiceActivityDetectionRequestMessage extends WorkerMessageBase {
+	messageType: 'VoiceActivityDetectionRequest'
+	input: string
```

Comment:
Fixed in https://github.com/echogarden-project/echogarden/pull/87/commits/97d3b1d5e2a0e407380002755bc11c7328e6b86a